amazonia/amz.py

```diff
@@ -63,11 +68,15 @@ def main():
     """
     template_file_path = args.template
     template_trop = create_stack(stack_input)
+    send_to_output = args.out
     template_data = template_trop.template.to_json(indent=2, separators=(',', ': '))
     with open(template_file_path, 'w') as template_file:
```

Comment:
@Layoric, This is a good enhancement, but we might want to avoid writing out to a file if the user wants amz to write to stdout instead.

Happy to merge this if you can expand your if/else to cover the `with open(template_file_path, 'w') as template_file:` chunk of code? (line 73 to 75) :)